addons/knowledge/static/src/components/permission_panel/permission_panel.js

```diff
@@ -87,6 +87,9 @@ class PermissionPanel extends Component {
         const index = this.state.members.findIndex(current => {
             return current.partner_id === session.partner_id;
         });
```

Comment:
on peut pas faire pareil que pour l'autre ?
const index = this.state.members.indexOf(member);

Reply:
The issue is that in this method we don't have the member as an argument...